src/components/Icon/LoadingIcon/LoadingIcon.less

```diff
@@ -0,0 +1,44 @@
+.lucid-LoadingIcon {
+	// simple rotation in IE11 because IE11 doesn't support svg animations
+	-ms-animation: rotate-animation linear infinite;
+
+	circle {
+		fill: none;
+		stroke-width: 12;
+		stroke-linecap: round;
+	}
+
+	&-spinner {
+		transform-origin: center;
+		animation: dash-animation linear infinite;
+		stroke: @color-primary;
+		stroke-dasharray: 150px 100px;
+	}
+
+	&-circle {
+		stroke: @color-gray;
+	}
+
+}
+
+@keyframes rotate-animation {
+	to {
+		transform: rotateZ(-360deg);
+	}
+}
+
+@keyframes dash-animation {
+	0% {
+		stroke-dashoffset: 0px;
```

Comment:
this is not just `0` because IE edge required the unit to be `0px` rather than just `0` or it will fail to animation `stroke-dashoffset`.